boilerplate/flyte/end2end/test_run.py

```diff
@@ -152,7 +148,6 @@ def run(
     )
     r = requests.get(manifest_url)
     parsed_manifest = r.json()
-    workflow_groups = []
```

Comment:
why are we removing this?

Reply:
I noticed that the workflow_groups parameter is immediately reassigned in the next line, so I just think it might not need to be repeated.